README.md

```diff
@@ -1,5 +1,7 @@
 Seeker 1.0 Beta 6
```

Comment:
Please remove the version from the title and use markdown formatting to make it a proper heading.